tools/lib/language.js

```diff
@@ -63,8 +63,7 @@ class Language {
   }
 
   static fromFile(filename) {
-    if (filename.startsWith("/") || filename.startsWith("."))
-    {
+    if (filename.startsWith("/") || filename.startsWith(".") || process.platform === "win32") {
```

Comment:
This doesn't look correct though, what exactly is the problem you were trying to solve here?

Reply:
Windows file paths start with `C:\Users\...`  - without this platform check the file name becomes: 

```
/src/languages/C:\Users\...
```

Reply:
Ah, that's not a correct fix though, that will cause it to break elsewhere.

You'd need to do something like this:

https://stackoverflow.com/questions/21698906/how-to-check-if-a-path-is-absolute-or-relative